Gemfile

```diff
@@ -3,6 +3,9 @@ source 'https://rubygems.org'
 # sufia stuff!
 gem 'sufia', '7.3.0'
 gem 'blacklight_range_limit', ">= 6.1.2"
+# It may seem like this is a dependency upstream, but we need it here.
+# https://project-hydra.slackarchive.io/dev/page-100/ts-1491587437169731
+gem 'rsolr', '~> 1.0'
```

Comment:
this slack link sadly won't work, because hydra slack only keeps limited history since it's a free slack account. It would be nice to able to link to slack conversations in comments! There's some slack archive service whose URLs you can use as a workaround, but too much trouble for me too.